### PR TITLE
Note explaining why MPAs are more accessible

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- New or updated content

#### Description

This PR adds a small note explaining why MPAs have better accessibility than SPAs.

### CODE REVIEW

**Comment by sarah11918** [comment]
Thanks for this, @Yan-Thomas This is short, and sweet, and I *might* even be willing to say a little more?

I did a quick search, and found this comment: https://nolanlawson.com/2022/06/27/spas-theory-versus-practice/#comment-224514

Is there anything from here you think might be worth mentioning? I don't know whether it gets too much into the technicalities of what is actually happening, but I know I personally found it interesting to read to connect with the problem, and maybe a bit more detail in our explanation will be a bit more compelling? ("What? I didn't know JS events didn't send this to a screen reader??) I don't know. What do you think?

File: src/pages/en/concepts/mpa-vs-spa.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ When comparing MPAs vs SPAs, there is no "better" or "worse" choice. It all come
 
 **Astro prioritizes the performance and simplicity of MPAs because it makes the most sense for our usecase of content-focused websites.** More stateful, interaction-heavy websites may benefit more from the app-like architecture that SPAs bring at the expense of first-load performance.
 
+:::note[Accessibility]
+MPAs use the standard `<a>` element for navigation. This comes with the benefit of no overhead announcing route changes or managing focus between pages to improve your website's accessibility.
+:::
```
Comment (Yan-Thomas) [inline src/pages/en/concepts/mpa-vs-spa.md:58]:
```suggestion
:::note[Accessibility and UX]
MPAs use the standard `<a>` element for navigation. This comes with the benefit of letting the platform manage your route changes and focus between pages. In counterpoint, SPAs come with the overhead of making you manage all this by yourself. And even then, you don't have access to some of the platform's features like [SFNSP](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation-starting-point) and some events that improve MPAs' UX and accessibility.
:::
```

**Review by Yan-Thomas — COMMENTED** [review]
> Is there anything from here you think might be worth mentioning? I don't know whether it gets too much into the technicalities of what is actually happening, but I know I personally found it interesting to read to connect with the problem, and maybe a bit more detail in our explanation will be a bit more compelling?

I tried to make a good compromise between explaining and not being too technical, let me know what you think of my suggestion here. Loved the article and comment you linked :raised_hands:

File: src/pages/en/concepts/mpa-vs-spa.md [hunk 2]
```diff
@@ -53,6 +53,9 @@ When comparing MPAs vs SPAs, there is no "better" or "worse" choice. It all come
 
 **Astro prioritizes the performance and simplicity of MPAs because it makes the most sense for our usecase of content-focused websites.** More stateful, interaction-heavy websites may benefit more from the app-like architecture that SPAs bring at the expense of first-load performance.
 
+:::note[Accessibility]
+MPAs use the standard `<a>` element for navigation. This comes with the benefit of no overhead announcing route changes or managing focus between pages to improve your website's accessibility.
```
Comment (sarah11918) [inline src/pages/en/concepts/mpa-vs-spa.md:57]:
```suggestion
MPAs use the standard `<a>` element for navigation. This provides important accessibility features such managing focus states and announcing route changes by default.
```
Another take? @Yan-Thomas

Comment (Yan-Thomas) [reply src/pages/en/concepts/mpa-vs-spa.md:57]: Loved it!

Comment (sarah11918) [reply src/pages/en/concepts/mpa-vs-spa.md]: Oh good, and you even caught the missing word! ;)

**Comment by sarah11918** [comment]
LGTM, now @Yan-Thomas ! Thanks so much for this!

**Comment by Yan-Thomas** [comment]
Thanks for the review @sarah11918! 💜

**Pull request merged**